src/server.js

```diff
@@ -1,16 +1,21 @@
-import * as dotenv from 'dotenv';
-dotenv.config();
-
 import express from 'express';
 import cors from 'cors';
+import dotenv from 'dotenv';
 
-import routes from './routes.js';
+import oauthRoute from './routes/oauth.js';
+import studentsRoute from './routes/students.js';
+import usersRoute from './routes/users.js';
```

Comment:
Nomear no plural, já que são múltiplas rotas definidas no router.

```suggestion
import oauthRoutes from './routes/oauth.js';
import studentsRoutes from './routes/students.js';
import usersRoutes from './routes/users.js';
```